src/OrchardCore.Modules/OrchardCore.Liquid/README.md

```diff
@@ -215,6 +215,25 @@ The following properties are available on the `User` object.
 | `Identity.Name` | `admin` | The name of the authenticated user. |
 | `Identity.Claims` |  | The claims of the authenticated user. |
 
+
+#### User Filters
+##### authorize filter
+checks if the User has permission clearance, optionally on a resource 
+```liquid
+{{ User | authorize:"EditContent",Model.ContentItem }}
```

Comment:
has_authorization, has_permission? The idea is to start with a verb like the other ones